readme/Readme.scalatex

```diff
@@ -322,6 +335,89 @@
                     To explicitly isolate a cut to one branch of a parser, place that branch within @hl.scala{NoCut}.  Cuts within that branch will prevent backtracking inside that branch, but if that branch fails alternate branches will be tried as normal.
 
                 @hl.ref(tests/"ExampleTests.scala", Seq("'composenocut", ""))
+        @sect{Unapply}
+            @p
+                @hl.scala{Parser} class has (as many other classes in Scala) a convenient method @hl.scala{unapply}
+                which allows to do pattern matching using parsers (like regular expressions) in your code.
+
+                @hl.ref(tests/"ExampleTests.scala", "val capture1", "val captureOpt")
+
+
+    @sect{Writing Byte Parsers}
+        @p
+            In the same way as the @hl.scala{String} parsers we can build @hl.scala{Array[Byte]} or simply @hl.scala{Byte} parsers.
+        @p
+            The first example will be also about parsing a single byte sequence
+        @hl.ref(bytetests/"ByteTests.scala", Seq("'sequence", ""))
```

Comment:
I think two new concepts is not a problem for understanding :) Let's slightly rewrite it with adding comments for explanation of them.